src/components/Dashboard/ThisWeek/NewsletterMenu/MenuItems/ExportPhysical/ExportPhysical.tsx

```diff
@@ -31,8 +40,36 @@ const CloseButton = styled(IconButton)(({ theme }) => ({
   },
 }));
 
-const ExportPhysical = ({ handleClose }: Props): ReactElement<Props> => {
+const LabelButton = styled(Button)(({ theme }) => ({
+  width: '90%',
+  margin: theme.spacing(1, 0),
+}));
+
+const ExportPhysical = ({
+  handleClose,
+  accountListId,
+}: Props): ReactElement<Props> => {
```

Comment:
```suggestion
const ExportPhysical: React.FC<Props> = ({
  handleClose,
  accountListId,
}) => {
```

I think the React.FC handles the children case better and you don't have to specify a return type. Also the JSX you are returning doesn't take any props so `ReactElement<Props>` isn't accurate.

Reply:
Ah yep, will update!